src/IssueViz/Models/AnalysisIssueVisualizationConverter.cs

```diff
@@ -74,12 +66,7 @@ private IReadOnlyList<IAnalysisIssueLocationVisualization> Convert(IEnumerable<I
             var locationNumber = 1;
 
             var convertedLocations = locations
-                .Select(location =>
-                {
-                    var isNavigable = navigationChecker.IsNavigable(location);
-
-                    return new AnalysisIssueLocationVisualization(locationNumber++, isNavigable, location);
-                })
+                .Select(location => new AnalysisIssueLocationVisualization(locationNumber++, location))
```

Comment:
The navigationChecker was never used so I just removed it.
The navigability will be handled now by the buffer and the explicit check on selection.